README.md

```diff
@@ -149,16 +148,15 @@ configured by setting the `manager` option to `subscription_manager`:
 
```

Comment:
The vagrant-registration plugin uses `subscription_manager` as the default manager. This can also be explicitly configured by setting the `manager` option to `subscription_manager`: